components/domobject_derive/lib.rs

```diff
@@ -57,6 +57,14 @@ fn expand_dom_object(input: syn::DeriveInput) -> proc_macro2::TokenStream {
                 self.#first_field_name.init_reflector(obj);
             }
         }
+
+        impl #impl_generics Eq for #name #ty_generics #where_clause {}
```

Comment:
@sagudev sorry for delay, I moved them to domobject_derive ready for review again.